### PR TITLE
Logger backend

### PR DESCRIPTION
**Pull request by mrbaboon**

### CODE REVIEW

File: src/auditlog/backends/base.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                changes=changes,
+                **kwargs)
+
+    def log_update(self, instance, **kwargs):
+        """Handle model update logging"""
+        if instance.pk is not None:
+            try:
+                old = instance.__class__.objects.get(pk=instance.pk)
+            except instance.__class__.DoesNotExist:
+                pass
```
Comment (bujniewicz) [inline src/auditlog/backends/base.py:25]: Silent failure here.

File: src/auditlog/backends/base.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        if instance.pk is not None:
+            changes = model_instance_diff(instance, None)
+
+            self.create_log(
+                instance=instance,
+                action=LogEntry.Action.DELETE,
+                changes=changes,
+                **kwargs)
+
+    def create_log(self, action, instance, changes, **kwargs):
```
Comment (bujniewicz) [inline src/auditlog/backends/base.py]: Confusing names: log_create and create_log are too similar while both belong to public interface. Adding a leading underscore to create_log or a name change should help.

**Review by bujniewicz — CHANGES_REQUESTED** [review]

File: src/auditlog/backends/base.py (hunk 1 above)
Comment (mrbaboon) [reply src/auditlog/backends/base.py:25]: @bujniewicz have to leave this as its from their codebase and i don't want to break backwards compatibility.

File: src/auditlog/backends/base.py (hunk 2 above)
Comment (mrbaboon) [reply src/auditlog/backends/base.py]: I would tend to agree, but this way you can skip the signals and call `create_log` directly without having to go through model signals.

Comment (bujniewicz) [reply src/auditlog/backends/base.py]: So we can just change the name to be more meaningful and distinguishable.

File: src/auditlog/backends/base.py (hunk 1 above)
Comment (bujniewicz) [reply src/auditlog/backends/base.py:25]: ugh, this sucks :P

**Pull request merged**